packages/web-runtime/src/store/config.ts

```diff
@@ -37,6 +37,10 @@ const state = {
   options: {
     contextHelpers: true,
     defaultExtension: 'files',
+    editor: {
+      autosaveEnabled: false,
+      autosaveInterval: 120000
```

Comment:
Like I said earlier in a zoom call, I'm not a fan of configuring this in `ms`. I know that it was `consistency` vs `practicality`. Giving the admin the chance to let an editor autosave a file every 100 milliseconds doesn't sound like a good idea to me. But I get the `consistency` argument... no veto, but wanted to express my `meh`.